### ISSUE TITLE
Issue while Installing on stretch

### ISSUE THREAD

**Issue by Potjoe-97** [empierre/MyDomoAtHome#174]
Hi everyone. 

I just moved to Stretch version of Raspbian, and I have some troubles during installation process. 
Here is the error code : 
`Préparation du dépaquetage de .../mydomoathome_0.2.39_all.deb ...
/var/lib/dpkg/tmp.ci/preinst: ligne 4: npm : commande introuvable
dpkg: erreur de traitement de l'archive /var/cache/apt/archives/mydomoathome_0.2                     .39_all.deb (--unpack) :
 le sous-processus nouveau script pre-installation a retourné une erreur de sort                     ie d'état 127
Des erreurs ont été rencontrées pendant l'exécution :
 /var/cache/apt/archives/mydomoathome_0.2.39_all.deb
E: Sub-process /usr/bin/dpkg returned an error code (1)`

Also, when I tried to install the correct version of Npm from the repo, i received a 403 forbidden error code. I have moved to the Ssl version of the repo, I really don't know where does it come from. 

Thanks for your help

**Comment by Potjoe-97** [comment]
I managed to install nodejs and npm by downloading the file through a browser, as I said, wget command was returning a 403 error. 
But now, when I try sudo apt-get install MyDomoAtHome, i got an error saying that there is no such packet, whereas I added the repo in sources.list.

Edit : the error message is actually the following

```
Aucune version du paquet mydomoathome n'est disponible, mais il existe dans la base
de données. Cela signifie en général que le paquet est manquant, qu'il est devenu obsolète
ou qu'il n'est disponible que sur une autre source

E: Le paquet « mydomoathome » n'a pas de version susceptible d'être installée
```

Edit 2 : Weird... Managed to fix it by downloading the .deb directly from the repo. Apt-get woud'nt hear anything. Solved and closed.
